spec.go

```diff
@@ -38,12 +38,12 @@ type Interface interface {
 	// for it to start. A service account and cluster role binding are also created.
 	// As a first step, it checks if Tiller is already ready, in which case it
 	// returns early.
-	EnsureTillerInstalled(ctx context.Context) error
+	EnsureTillerInstalled(ctx context.Context, upgrading bool) error
```

Comment:
I would like to avoid naming dispute since these would eventually be deleted on helm v3.